src/Kernel/Behavior.class.st

```diff
@@ -17,6 +17,17 @@ Class {
 	#category : #'Kernel-Classes'
 }
 
+{ #category : #'simulation support' }
+Behavior class >> classOf: anObject [
+	"The class of anObject.
+	This is equivalent to `anObject class` but without a message to be send to `anObject`.
+	This can be very handy if the class of `anObject` is unreliable.
+	e.g. no inheritance from ProtoObject, or having a nil methodDict."
+
+	<primitive: 111>
+	self primitiveFailed
+]
```

Comment:
This method is already available in `MirrorPrimitives` and there was a previous effort to have it only once in that place (otherwise it was repeated in many places). I propose we remove this method and we use the other.

Reply:
MirrorPrimitives is undocumented and almost not used (only 4 references in the whole image outside of its own package `ReflectionMirrors-Primitives`, and those 4 are only for tests), its methods are also undocumented and non-discoverable.

Moreover, the class is not in Kernel, so it introduces an unwanted package dependency with ReflectionMirrors-Primitives.
Kernel should try to be minimal and self-contained. Some small code duplication is a lesser concern.

The version in Behavior is not that bad, as the class is meaningful for the intent.